### DISCUSSION TITLE
Homeschool student in New Jersey, How can I prove that I am homeschooled?

### DISCUSSION THREAD

**Discussion by cannotkenpute** [community/community#77366] [GitHub Education]
### Hi everyone,

hi everyone I am a home school student in New Jersey, I recently completed the Mercer County Community College GED preparation course, and I would love to have access to the GitHub student dev pack, can anyone help me?

### A suggestion...(optional)

homeschool

**Comment by hackelite01** [comment]
hey @goosecrash  , Yes you can apply for it. To clarify more kindly share the details. Drop the details on mail at [hackelite.sup@gmail.com](mailto:hackelite.sup@gmail.com)

**Comment by awirnr** [comment]
Response #1
